src/uproot/interpretation/objects.py

```diff
@@ -376,16 +377,14 @@ def simplify(self):
         ``self``.
         """
         if self._branch is not None:
-            try:
+            with contextlib.suppress(CannotBeStrided):
```

Comment:
This one is new to me and expresses the intent very well!

Reply:
Python 3.4+